### PR TITLE
Update default.nix requirements & version

### PR DESCRIPTION
**Pull request by arneturuc**
Update dependencies in accordance to `requirements.txt`

### CODE REVIEW

**Comment by arneturuc** [comment]
@joachimschmidt557 do you think we can automate this as well using Dependabot?
Else I would update it when reviewing a Dependabot Pull request

File: default.nix [hunk 1]
```diff
@@ -16,17 +16,16 @@ let
...
 mach-nix.buildPythonApplication {
   pname = "timewsync";
-  version = "1.0.0";
+  version = "1.1";
 
   src = ./.;
 
   format = "pyproject";
 
   requirements = ''
```
Comment (joachimschmidt557) [inline default.nix:25]:
```suggestion
  requirements = builtins.readFile ./requirements.txt;
```

**Review by joachimschmidt557 — COMMENTED** [review]
`mach-nix` is not ideal, maybe we can look into something better in future (*cough* Poetry). For now, let's eliminate the redundancy here.

**Review by joachimschmidt557 — APPROVED** [review]
LGTM

**Pull request merged**